archive-press.php

```diff
@@ -0,0 +1,326 @@
+<?php
+get_header();
+
+$current_year_id = IGV_get_option('_igv_site_options', '_igv_current_fair_year');
+$current_year = (!empty($current_year_id)) ? get_term($current_year_id)->slug : false; 
+// if we have the term ID, get the slug, 
+// otherwise set it false for later conditionals
+
+$press_page_text = IGV_get_option('_igv_page_options', '_igv_press_page_text');
+$press_page_image_id = IGV_get_option('_igv_page_options', '_igv_press_page_image_id');
+$press_accreditation = IGV_get_option('_igv_page_options', '_igv_press_accreditation_text');
+
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+  <!-- main posts loop -->
+  <section id="press-intro" class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col col-l col-l-12">
+          <h1>
+          <?php 
+            _e('[:en]Press[:es]Prensa;'); 
+          ?>
+          </h1>
+        </div>
+      </div>
+<?php 
+if (!empty($press_page_text) || !empty($press_page_image_id)) { 
+?>
+      <div class="row">
+<?php 
+  if (!empty($press_page_image_id)) {
+?>
+        <div class="col col-l col-l-6">
+          <?php echo wp_get_attachment_image($press_page_image_id, 'col-6'); ?>
+        </div>
+<?php 
+  }
+
+  if (!empty($press_page_text)) { 
+?>
+        <div class="col col-l col-l-6">
+          <div class="font-size-h3">
+            <?php echo apply_filters( 'the_content', $press_page_text ); ?>
+          </div>
+          <div class="font-size-h4">
+            <?php echo (!empty($press_accreditation) ? apply_filters( 'the_content', $press_accreditation ) : ''); ?>
+          </div>
+        </div>
+<?php 
+  }
+?>
+      </div>
+<?php 
+}
+?>
+    </div>
+  </section>
+
+<?php 
+$args = array (
+  'post_type'       => 'press',
+  'posts_per_page'  => '2',
+  'meta_key'        => '_igv_press_date',
+  'orderby'         => 'meta_value_num',
+  'meta_query'      => array( 
+    array(
+      'key' => '_igv_press_highlight',
+      'value' => 'on',
+    ),
+    array(
+      'key' => '_igv_press_url',
+    ),
+  ),
+);
+$highlight_press = new WP_Query( $args );
+
+if ( $highlight_press->have_posts() ) { 
+  $highlight_post = 0;
+?>
+  <section id="press-highlights" class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col col-l col-l-12 text-align-center">
+          <h2><?php _e('[:en]Highlights[:es]Destacados'); ?></h2>
+        </div>
+      </div>
+      <div class="row">
+<?php 
+    while( $highlight_press->have_posts() ) {
+      $highlight_press->the_post();
+
+      $highlight_meta = get_post_meta($post->ID);
+
+      $publication = $highlight_meta['_igv_press_publication'][0];
+      $date = $highlight_meta['_igv_press_date'][0];
+      $author = $highlight_meta['_igv_press_author'][0];
+      $link = $highlight_meta['_igv_press_url'][0];
+
+      if ($highlight_post == 0) { // Image first
+?>
+        <div class="col col-l-12 flex-row align-center">
```

Comment:
for things like this I think using partials for different post layouts is likely to be most replicable, readable and maintainable